meetings/mit2017/index.md

```diff
@@ -1,27 +1,28 @@
 ---
 layout: common
+redirect_from: "/developersmeetup/"
 ---
 
-# JuMP Developers Meetup/Workshop
+# First Annual JuMP-dev Workshop
```

Comment:
I don't know if we should change the name retroactively, or if we do at least leave a note of the original name.